fe/src/components/Header/Header.tsx

```diff
@@ -1,21 +1,20 @@
-import { Box, Flex, useColorModeValue } from '@chakra-ui/react';
+import { Flex } from '@chakra-ui/react';
 import { HeaderLeftSide } from './HeaderLeftSide';
 import { HeaderRightSide } from './HeaderRightSide';
 
 export default function Header(): JSX.Element {
-  const me = 'Peeter';
+  const me = undefined;
```

Comment:
`me` or current user should probably be stored in some outer state (perhaps some global state) similiar to `HomeContext`, currently passing it as prop to elements probably isn't the best solution. 🤔 

But that's mostly for future reference anyway. Maybe at least could add a `@todo` comment here as a reminder?